src/components/common/navigation/navigation.scss

```diff
@@ -256,6 +256,10 @@
     left: 68px;
 }
 
+.help-grid__feedback {
+    grid-template-rows: 36px 36px 1px 36px 36px 1px 36px 36px auto !important;
```

Comment:
Do we need `!important` here?

Reply:
Yes as .help-card will override this one